packages/hdwallet-vultisig/src/cosmos.ts

```diff
@@ -0,0 +1,40 @@
+import { StdTx } from "@cosmjs/amino";
+import { SignerData } from "@cosmjs/stargate";
```

Comment:
Let's add these to hdwallet-vultisig's package.json

Will magically work at build time because this is a monorepo and they're hoisted, but wouldn't automagically work for clients which is v bad